code/modules/mob/living/simple_animal/hostile/goose.dm

```diff
@@ -70,7 +70,7 @@
 /mob/living/simple_animal/hostile/retaliate/goose/proc/feed(obj/item/suffocator)
 	if(stat == DEAD || choking) // plapatin I swear to god
 		return FALSE
-	if(suffocator.custom_materials && suffocator.custom_materials[SSmaterials.GetMaterialRef(/datum/material/plastic)]) // dumb goose'll swallow food or drink with plastic in it
+	if(suffocator?.has_material_type(/datum/material/plastic)) // dumb goose'll swallow food or drink with plastic in it
```

Comment:
why the ?. ? goose code ~~kill me~~ checks for the input atom quite often, did you see a runtime here before?

Reply:
I suppose it isn't needed. I think I put that in on autopilot.

Reply:
Done.